subsys/bluetooth/host/gatt.c

```diff
@@ -2090,7 +2099,226 @@ void bt_gatt_disconnected(struct bt_conn *conn)
 	BT_DBG("conn %p", conn);
 	bt_gatt_foreach_attr(0x0001, 0xffff, disconnected_cb, conn);
 
+#if defined(CONFIG_BT_SETTINGS)
```

Comment:
`if (IS_ENABLED(CONFIG_BT_SETTINGS) && bt_addr_le_is_bonded()) { ...` ?

Reply:
Will fix it.